.jscsrc

```diff
@@ -17,7 +17,6 @@
     "fileExtensions": [".js"],
     "extract": ["*.html"],
     "maxErrors": null,
-    "verbose": true,
```

Comment:
verbose is true by default (according to a warning by JSCS when I ran it)
so we can remove this.

On Tue, May 17, 2016 at 9:40 PM, Sean Lip <notifications@github.com> wrote:

> In .jscsrc <https://github.com/oppia/oppia/pull/1849#discussion_r63644140>
> :
>
> > @@ -17,7 +17,6 @@
> >      "fileExtensions": [".js"],
> >      "extract": ["*.html"],
> >      "maxErrors": null,
> > -    "verbose": true,
>
> why?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/oppia/oppia/pull/1849/files/47fe9d59a729b2f80621a3fbfccb24da1c7466ee#r63644140>
>